### PR TITLE
fix(GraphQL): update parseType signature

### PR DESCRIPTION
**Pull request by j**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/graphql/graphql-js/blob/v0.13.0/src/language/parser.js#L166
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

**Review by IvanGoncharov — APPROVED** [review]

**Comment by j** [comment]
Still waiting for the merge! :( @IvanGoncharov

**Review by intellix — APPROVED** [review]

**Comment by IvanGoncharov** [comment]
> Still waiting for the merge! :( @IvanGoncharov

@j Sorry, can't help since I don't have commit rights to this repo.
I believe we need to wait for someone from Microsoft to merge it.

**Comment by DanielRosenwasser** [comment]
Sorry about the wait, thanks for the PR!

**Pull request merged**
